setup.py

```diff
@@ -18,6 +18,7 @@
     LONG_DESCRIPTION = f.read()
 
 INSTALL_REQUIRES = ['SALib==1.2',  # last version to work with python2
+                    'coverage',
```

Comment:
I think we can remove this dependency if we remove coverage from the code itself - any dev wanting to run coverage can just pip install it. btw: did you see pycharm also has coverage functions? right next to the profiling thing. did you check those out?

Reply:
Oh I've seen that before but have always thought that coverage needs to be installed to run it. I didn't try it before.

Reply:
yes, i think coverage needs to be installed first - i haven't run it either before - but still: that's not something the cea _requires_ so we don't need it in the INSTALL_REQUIRES.